doc/src/fix_propel_self.rst

```diff
@@ -8,94 +8,165 @@ Syntax
 
 .. parsed-literal::
 
-   fix ID group-ID propel/self mode magnitude keyword values ...
+   fix ID group-ID propel/self magnitude keyword values
```

Comment:
sorry, but this change should be undone. the "mode" keyword should remain at the location where is currently is (for compatibility) since it is required and not a "keyword/value" pair as is intended for the additional arguments.

Reply:
Okay! I have done as you requested.